tests/testthat/test_initCrit.R

```diff
@@ -0,0 +1,19 @@
+test_that("infill crit initialization", {
+
+  ps = makeParamSet(
+    makeNumericParam("x1", lower = -2, upper = 1),
+    makeIntegerParam("x2", lower = -1, upper = 2)
+  )
+  
+  expect_equal(crit.cb1, initCrit(crit.cb, ps))
```

Comment:
as observed above. please add a test that checks that initCrit does not overwrite a specified criterion